mpi-proxy-split/mpi-wrappers/mpi_request_wrappers.cpp

```diff
@@ -167,9 +167,13 @@ USER_DEFINED_WRAPPER(int, Testany, (int) count,
   MPI_Status *local_status = status;
 
   int retval;
-  *local_flag = 0;
+  *local_flag = 1;
   *local_index = MPI_UNDEFINED;
   for (int i = 0; i < local_count; i++) {
+    if (local_array_of_requests[i] == MPI_REQUEST_NULL) {
+      continue;
+    }
+    *local_flag = 0;
```

Comment:
In principle, we should not need the line above (setting `*local_flag = 0;`).  I'd feel better if you can remove this line, and run it again to see why it's needed.  Logically, this should not change the result, since `MPI_Test` must also set `*local_flag`.  And it's dangerous to commit code that we don't understand.

Reply:
I re-ran the test and it works without this line - I probably made a mistake in the prior tests.